jobs/geo/core.py

```diff
@@ -1,44 +1,90 @@
 from __future__ import annotations
 
-from io import StringIO
-
 from jobs.utils.snowflake import gdf_to_snowflake, snowflake_connection_from_environment
 
 
-def load_geo_data(conn, url: str, table: str) -> None:
-    """Load general geospatial data. Given a URL, load geospatial data into Snowflake."""
-    import geopandas
-    import requests
+def load_ca_geo_data(conn, year: str) -> None:
+    """Load CA Census geo data into Snowflake."""
+    from pygris import (
+        block_groups,
+        blocks,
+        counties,
+        county_subdivisions,
+        places,
+        primary_secondary_roads,
+        pumas,
+        tracts,
+    )
+
+    print(f"Downloading data for CA in year {year}")
+
+    state = "CA"
+    cache = True
+    dfs = []
+
+    county = counties(state=state, year=year, cache=cache)
+    county["year"] = year
```

Comment:
I'm curious the logic of adding a `year` column to all of these. I don't necessarily object, but I'm not sure it's necessary either.

Reply:
I just think it's helpful to have another datapoint from a UX POV on what year of census data you are working with. If you think it's truly unnecessary I will remove.

Reply:
Going to remove for now, can re-add if we think a good idea. Done